appinventor/buildserver/build.xml

```diff
@@ -71,6 +71,21 @@
       <!-- Component/extension assets (for testing) -->
       <fileset dir="${appinventor.dir}/components/src" includes="**/assets/*"/>
     </copy>
+
+    <!-- Copy natives libray -->
+    <copy todir="${classes.files.dir}/armeabi" flatten="true">
```

Comment:
I'd rather we copy the libraries to `${public.deps.dir}` and do this copy in components/build.xml along with the JAR component of the library. Libraries (both Java and native) should live under lib/, not be placed in components/src/.